js/mexc.js

```diff
@@ -30,6 +30,7 @@ module.exports = class mexc extends Exchange {
                 'cancelOrder': true,
                 'createMarketOrder': false,
                 'createOrder': true,
+                'createReduceOnlyOrder': false,
```

Comment:
I definately think we can create this one, it's just like `createOrder` but you might need to pass in a reduceOnly argument, it's kind of needed for contract trading

Reply:
I'm not seeing any evidence of reduce only orders being supported on mexc after looking through the api docs and the trading interface but let me know if you can find something

Reply:
I thought a reduce-only order was required to close a contract position?

Reply:
From my understanding, reduce only is used so that there's no risk of accidentally opening a new position when trying to close or reduce a position and is only supported on some exchanges. On mexc you can specify the side argument with close short or close long to close the contract position

Reply:
So I think that would be equivalent to a reduceOnly order

@kroitor @frosty00 ?

Reply:
I don't support the idea of having `createReduceOnlyOrder` up until we add and unify the support for a `reduceOnly` param in all `createOrder` implementations where available from the exchange, and then we can move `createReduceOnlyOrder` to the base class. But I'm not against it being there as `false` either.

Reply:
> I don't support the idea of having `createReduceOnlyOrder` up until we add and unify the support for a `reduceOnly` param in all `createOrder` implementations where available from the exchange, and then we can move `createReduceOnlyOrder` to the base class. But I'm not against it being there as `false` either.

It being false means that it cannot be implemented. Mexc allows you to close an trade, and I understand that as being the same as placing a reduce order with the amount being the full amount of the trade

Reply:
I think that it's similar but not really the same, all exchanges let you close the order but not all of them have reduce only orders and many of them seem to add it as a feature later on. Exchanges that have reduce only functionality let you set the side and reduceOnly option separately and have a button to enable it in the interface similar to setting a stop loss, Mexc doesn't have that. I think we could make separate methods for reduceOnlyLong and reduceOnlyShort setting the side to 'close long' or 'close short' or do some handling in the createOrder method but I'm not sure that it's needed or if it would be technically correct in this situation. I think it's possible that mexc adds actual reduce only functionality in the future

Reply:
It seems like exchanges might handle reduceOnly orders quite differently. For ascendex I just had to set execInst: 'reduceOnly'. But I found this https://twitter.com/okx/status/1466044664083980289 And Okex is similar to mexc in that they allow 'close long' and 'close short' orders but don't have a reduce only button on the interface and nothing related to reduce only in the api docs.

Reply:
@Dan-krm i think OKEX does have a reduceOnly feature in their API:

<img width="1428" alt="Screenshot 2022-01-29 at 17 06 18" src="https://user-images.githubusercontent.com/1294454/151664107-7b898601-5106-4980-9b87-6a878908e832.png">

Most of the exchanges will allow reduce only orders with a specific field+value sent in the request along with the other order parameters upon creating it. To me the unification of reduceOnly begins with adding the unified support for that flag to createOrder implementations first and foremost. Then we can add the syntax sugar like createReduceOnlyOrder and other frequently-asked methods.

Reply:
Thanks for the correction, I must've been looking at their old api docs